File: mikan.c
#include <stdio.h>

int n;

int max(int a, int b) {
	return a > b ? a : b;
}

int mikan(int w) {
	return max((w + n / 2) / n, 1) * n;
}

int main() {
	int m;
	scanf("%d %d", &n, &m);
	while (m--) {
		int w;
		scanf("%d", &w);
		printf("%d\n", mikan(w));
	}
	return 0;
}

File: mikan.js
const lines = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [n, m] = lines[0].split(" ").map(Number)

function mikan(w) {
    return Math.max(Math.round(w / n), 1) * n;
}

for (var i = 1; i <= m; i++) console.log(mikan(Number(lines[i])));